### ISSUE TITLE
Enable versioning for gcp_scanner

### ISSUE THREAD

**Issue by mshudrak** [google/gcp_scanner#20]
There is no version history for GCP Scanner. We need to evaluate what options Github offer and/or develop our own versioning schema for the scanner.

**Comment by ArkaprabhaChakraborty** [comment]
I want to take this up :)

**Comment by SaxenaAnushka102** [comment]
Hi @mshudrak!! Could you please elaborate a bit more about what type of versioning schema is expected (like branch-based or tag based or some other type)? I'd like to contribute, please could you assign the issue to me?

**Comment by ZhengLin-Li** [comment]
Hi @mshudrak I am wondering if we could use the Github release to do this?

**Comment by HarshitAgarwal22000** [comment]
Hey @mshudrak, would love to work on this and suggest possible solutions which can help us in this specific issue :)

**Comment by mshudrak** [comment]
Hey folks, thanks for interest. We have multiple people want to work on this issue. Please go ahead and work on it if you want. Github release sounds good for me. PTAL at https://github.com/DynamoRIO/dynamorio/ versioning as an inspiration.

**Comment by sriharshitha07** [comment]
Hello, @mshudrak!! Could you perhaps specify on the kind of versioning schema required ,like tag or branch based?  I'd would really like to partake; could you please allocate the issue to me?

**Comment by mshudrak** [comment]
Hi @sriharshitha07, we are looking for tag based schema. There is ongoing PR on this one, if you want you can help @ZhengLin-Li on that. We just don't want any third-party depencendcy.

**Comment by sriharshitha07** [comment]
> Hi @sriharshitha07, we are looking for tag based schema. There is ongoing PR on this one, if you want you can help @ZhengLin-Li on that. We just don't want any third-party depencendcy.

alright ..thankyou

**Comment by sumit-158** [comment]
Hi, @mshudrak What about using [release-please-action](https://github.com/marketplace/actions/release-please-action), It's good I have used it [here](https://github.com/openfoodfacts/facets-knowledge-panels/blob/main/.github/workflows/release-please.yml)? What your thoughts on this?

**Comment by mshudrak** [comment]
Yep, @sumit-158 this is the project that we can use for release actions.

**Comment by mshudrak** [comment]
There are two ongoing PRs but both of them are using third-party projects that I would prefer to avoid linking the scanner with.

**Comment by sumit-158** [comment]
@mshudrak Open the third Pr :-)

**Comment by mshudrak** [comment]
Ok, right now we have five PR requests related to this issue #34, #35, #98, #100, #102. Please take a look at previous PRs before considering submitting a new PRs to this one.

**Comment by mshudrak** [comment]
TL;DR: We do not want to use any non-maintained or untrusted third-party Github actions as I commented on #34, #35, #98, #102. We will proceed forward with Github Actions plugin offered in #100.

**Comment by anantyadav0** [comment]
This project is similar to my course curriculum, if this project is given to me then i surely  give my best. And this project will helpful for my proffesional future.

**Comment by mshudrak** [comment]
It is done. We will rely on Github Actions to make releases. We will do them manually once a large set of features are implemented. PyPI workflow will be triggered automatically on new release.
